src/pubmed_source/index.js

```diff
@@ -0,0 +1,18 @@
+const { logger } = require('../logging');
+const { rid } = require('../graphkb');
+const { pubmed: SOURCE_DEFN } = require('../sources');
+
+const uploadFile = async ({ conn }) => {
+    logger.info('retrieve the record details');
+
+    try {
+        const source = rid(await conn.addSource(SOURCE_DEFN));
+        logger.log('info', `pubmed source: ${source}`);
+    } catch (err) {
+        logger.error(err);
+    }
+};
+
+module.exports = {
+    SOURCE_DEFN: {}, uploadFile,
+};
```

Comment:
A whole new loader just to add the Pubmed Source record :thinking:
Could we instead have a source loader that load src/sources.js :man_shrugging: ?

Reply:
I don't see why not? I'll change it to do that instead